### PR TITLE
feat(content): #35 Implement Spotlight component

### PR DESCRIPTION
**Pull request by rlr**

### CODE REVIEW

File: content-src/components/Base/Base.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ const {actions} = require("actions/action-manager");
 const Header = require("components/Header/Header");
 const TopSites = require("components/TopSites/TopSites");
 const ActivityFeed = require("components/ActivityFeed/ActivityFeed");
+const Spotlight = require("components/Spotlight/Spotlight");
+
+// TODO: replace this with the appropriate actions/reducers/shim.
+const fakeSpotlightItems = require("lib/shim").data.fakeSpotlightItems;
```
Comment (rlr) [inline content-src/components/Base/Base.js]: I wasn't sure if we were going to need new actions or reuse the existing ones so just did this for now...

**Comment by rlr** [comment]
not sure why travis isn't running on this PR

**Comment by k88hudson** [comment]
I think it might be backed up at the moment

File: content-src/components/Spotlight/Spotlight.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
+.spotlight {
```
Comment (k88hudson) [inline content-src/components/Spotlight/Spotlight.scss]: The overflow-y: auto on .new-tab-wrapper is cutting off the bottom of the shadows on these, can you just remove that rule from `NewTabPage/NewTabPage.scss`? We don't want it to scroll anyway i think

File: content-src/components/Spotlight/Spotlight.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    a {
+      color: $dark-grey;
+    }
+  }
+
+  .spotlight-info {
+    padding: 10px;
+  }
+
+  .spotlight-description {
```
Comment (k88hudson) [inline content-src/components/Spotlight/Spotlight.scss]: i think this would be better at font size 14px

Comment (rlr) [inline content-src/components/Spotlight/Spotlight.scss]: btw, is there a way to have like `text-overflow: ellipses;` work on multiline text like this? So we could have max 3 lines or something

Comment (k88hudson) [inline content-src/components/Spotlight/Spotlight.scss]: unfortunately not :(

**Comment by k88hudson** [comment]
R+ after those changes and a rebase. Note that the tests are failing now due to unrelated reasons (olivier is fixing currently)

**Pull request closed without merging**